README.md

```diff
@@ -1,2 +1,77 @@
-# pyclient
-Python client for interacting with the AllOfUs workbench API.
+# Python client for All of Us Workbench Jupyter Notebooks
+
+The client uses Python 2.7, to match FireCloud Jupyter environments.
+
+## Local testing
+
+### Configure
+
+The file all_of_us_config.json contains configuration settings used during
+local tests; this file is copied to notebook clusters during localization.
+
+If you want to point at a different workspace or CDR, you can change the values;
+check in your changes if you want it to be permanent.  
+
+### Install
+
+```Shell
+virtualenv venv
+. venv/bin/activate
+
+
+### Regenerate the Swagger client
+
+```Shell
+pyclient$ ./project.rb swagger-regen
+```
+
+This will place generated Python files in pyclient/py/aou_workbench_client/swagger_client,
```

Comment:
nit: I wouldn't include "pyclient" in these paths, the reader can only assume paths are relative to the current dir, or the repo root (same thing in this case). Also it further assumes a directory name used by the caller for this repo (this doesn't necessarily need to match the repo name, though usually does).

Reply:
Done.